modules/fingerprint/tests/testRun/resinhup.conf

```diff
@@ -1,5 +1,9 @@
 [FingerPrintScanner]
-defaultFingerPrintFile: /resinOS-fingerprints
-whitelist:
-     ./fingerprint/tests/testRun/tree/dir3
-     ./fingerprint/tests/testRun/tree/dir2/file3
+
+root_defaultFingerPrintFile: resin-root.fingerprint
+root_whitelist:
+    ./fingerprint/tests/testRun/tree/dir3
```

Comment:
Shouldn't this be: fingerprint/tests/testRun/root_tree/dir3 or something like that ?